hack/verify-govet.sh

```diff
@@ -5,7 +5,7 @@ set -o pipefail
 
 GO_VERSION=($(go version))
 
-if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.4') ]]; then
+if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.5') ]]; then
```

Comment:
flags have changed for `go tool vet` and I think we can migrate testing to `1.5.*`